packages/bcer-api/app/src/location/location.service.ts

```diff
@@ -351,4 +353,25 @@ export class LocationService {
       .getMany();
       return { ...saleReportYear, data: locations };
   }
+
+  async cancelAllLocationWithExpiredNOI(): Promise<UpdateResult>{
+    // TODO: Add location != 'deleted'
```

Comment:
Hopefully we won't need this if we end up using the built in `softDelete` TypeORM functionality for deleting a location 🙃  (TypeORM *should* exclude it from queries automatically)